consul-k8s-federated/variables.hcl

```diff
@@ -4,7 +4,7 @@ variable "consul_k8s_module" {
   local path for testing.
   EOT
 
-  default = "github.com/shipyard-run/blueprints?ref=891c937844bdbec673f5428b1a3c8bff4e207727/modules//kubernetes-consul"
+  default = "github.com/shipyard-run/blueprints/modules//kubernetes-consul"
```

Comment:
Usually we pin modules so that when a module changes it does not break any downstream dependencies. The off set to this is as you found, things get changed upstream and they are not updated into downstream modules.